hoodie-common/src/main/avro/HoodieCompactionOperation.avsc

```diff
@@ -0,0 +1,71 @@
+{
+   "namespace":"com.uber.hoodie.avro.model",
+   "type":"record",
+   "name":"HoodieCompactionWorkload",
+   "fields":[
+      {
+         "name":"compactorId",
+         "type":["null","string"]
+      },
+      {
+         "name":"operations",
+         "type":["null", {
+            "type":"array",
+            "items":{
+               "name":"HoodieCompactionOperation",
+               "type":"record",
+               "fields":[
+                  {
+                     "name":"dataFileCommitTime",
+                     "type":["null","string"]
+                  },
+                  {
+                     "name":"dataFileSize",
+                     "type":["null","long"],
+                     "default": null
+                  },
+                  {
+                     "name":"deltaFilePaths",
+                     "type":["null", {
+                        "type":"array",
+                        "items":"string"
+                     }],
+                     "default": null
+                  },
+                  {
+                     "name":"dataFilePath",
+                     "type":["null","string"],
+                     "default": null
+                  },
+                  {
+                     "name":"fileId",
+                     "type":["null","string"]
+                  },
+                  {
+                     "name":"partitionPath",
+                     "type":["null","string"],
+                     "default": null
+                  },
+                  {
+                     "name":"metrics",
```

Comment:
what is the use-case for this ?

Reply:
Used by Stats collection during compaction phase